src/core/Form/Radiobutton/Radiobutton.baseStyles.tsx

```diff
@@ -0,0 +1,159 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { disabledCursor } from '../../../components/utils/css';
+import { element, font } from '../../theme/reset';
+import { focus } from '../../theme/utils/focus';
+
+/* stylelint-disable no-descending-specificity */
+export const baseStyles = withSuomifiTheme(
+  ({ theme }: TokensAndTheme) => css`
+    ${element({ theme })}
+    ${font({ theme })('bodyText')}
+
+  &.fi-radiobutton {
+    & .fi-radiobutton_hintText {
+        display: block;
+        padding-left: 26px;
+        color: ${theme.colors.depthDark1};
+        ${theme.typography.bodyTextSmall};
+    }
+
+    & .fi-radiobutton_input {
+      opacity: 0;
+      position: absolute;
+      margin: 8px 0 0 3px;
+
+      + label {
+        position: relative;
+        display: inline-block;
+        cursor: pointer;
+        min-height: 27px;
+        line-height: 1.5em;
+        padding-left: 26px;
+
+        &::before {
```

Comment:
Some comments for neat css drawing might help in the future. E.g. radio-input circle etc.

Reply:
Added comments to explain all those pseudo codes of radio input circles in the recent commit.